data/templates/snom.tmpl

```diff
@@ -703,11 +703,11 @@
         <xsi_max_anywhere_portals perm="">10</xsi_max_anywhere_portals>
         <xfer_dest_order_lifo perm="">on</xfer_dest_order_lifo>
         <remote_blacklist_action_timer perm="">1</remote_blacklist_action_timer>
-        <prov_polling_enabled perm="">off</prov_polling_enabled>
-        <prov_polling_mode perm="">rel</prov_polling_mode>
-        <prov_polling_period perm="">0</prov_polling_period>
+        <prov_polling_enabled perm="">{{ provisioning_freq == 'never' ? 'off' : 'on' }}</prov_polling_enabled>
+        <prov_polling_mode perm="">random</prov_polling_mode>
+        <prov_polling_period perm="">86400</prov_polling_period>
         <prov_polling_time perm="">00:00</prov_polling_time>
-        <prov_polling_time_rand_end perm="">00:00</prov_polling_time_rand_end>
+        <prov_polling_time_rand_end perm="">05:00</prov_polling_time_rand_end>
```

Comment:
Please set 01:00-04:00, IIRC Gigaset or some other brand uses that time interval.